internal/controller/vrg_volrep_test.go

```
@@ -2515,11 +2515,20 @@ func (v *vrgTest) kubeObjectProtectionValidate() *ramendrv1alpha1.VolumeReplicat
 }
 
 func kubeObjectProtectionValidate(tests []*vrgTest) {
+	if len(tests) == 0 {
+		return
+	}
```

This is not related to the fix, split to another commit and explain why this is needed.

---

We will not intentionally call this with empty tests. But what if somehow we end up with an empty tests slice when we're populating it in the test functions? Will it not be safe to have a base condition to handle that? 
And it will not happen when we're running the whole test suite, but when we're trying to run some specific tests, we may end up with an empty tests slice..?

And in the test files, some test cases are created conditionally

Still if you feel its not required for now, I can push it out..

---

If we call this with empty test list this may be a bug and better fail instead of silently returning. I'm sure this is not required for this change.